anchore_engine/clients/skopeo_wrapper.py

```diff
@@ -171,6 +194,47 @@ def download_image(fulltag, copydir, user=None, pw=None, verify=True, manifest=N
 
     return True
 
+def get_digest_value(digest_with_algorithm_prefix: str):
+    digest_raw = digest_with_algorithm_prefix
+
+    if ":" in digest_raw:
+        _, digest_value = digest_raw.split(":", 1)
+        return digest_value
+
+    return digest_raw
+
+def ensure_no_nondistributable_media_types(oci_index_file_path: str):
+    manifest_file_path = get_manifest_path_from_index(oci_index_file_path)
+
+    with open(manifest_file_path, 'r') as manifest_file:
+        data = manifest_file.read()
```

Comment:
Since `data` is later loaded into JSON I believe you want to load it here with `json.load()` https://docs.python.org/3/library/json.html#json.load